test/rules/src/https_everywhere_checker/http_client.py

```diff
@@ -325,6 +327,9 @@ def staticFetch(url, options, platformPath):
 			c.setopt(c.SSLVERSION, options.sslVersion)
 			c.setopt(c.VERBOSE, options.curlVerbose)
 			c.setopt(c.SSL_CIPHER_LIST, options.cipherList)
+			if urlParts.hostname[-6:] == '.onion':
+				c.setopt(c.CURLOPT_PROXY, '127.0.0.1:9050')
```

Comment:
please replace line 331 and 332 with the following 

```python
c.setopt(c.PROXY, 'socks5://127.0.0.1:9050')
```
or 
```python 
c.setopt(c.PROXY, '127.0.0.1')
c.setopt(c.PROXYPORT, 9050)
c.setopt(c.PROXYTYPE, c.PROXYTYPE_SOCKS5_HOSTNAME)
```

this typo is so difficult to trace...